src/cli/command-package.ts

```diff
@@ -29,8 +29,18 @@ export default async function run(options: CliOptions) {
 
   // Ok, we're updating a package in a directory not under our control.
   // We need to be sure it's safe.
-  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s']);
-  if (!options.force && (stdout || stderr)) {
+  let stdout, stderr;
+  try {
+      ({stdout, stderr} = await exec(inDir, 'git', ['status', '-s']));
+  } catch (e) {
+      ({stdout, stderr} = e);
```

Comment:
why this? What Errors have `stdout` and `stderr` on them?

Reply:
https://github.com/Polymer/polymer-modulizer/blob/master/src/util.ts#L84
Normally when the Promise fails (when for example the dir is not controlled by git) it is unhandled and error object has those properties.